elasticdl/python/common/args.py

```diff
@@ -403,15 +403,22 @@ def add_common_args_between_master_and_worker(parser):
         "--model_params",
         type=str,
         default="",
-        help="The dictionary of model parameters in a string that will be "
-        'used to instantiate the model, e.g. "param1=1,param2=2"',
+        help="The model parameters in a string separated by semi-colon "
+        'used to instantiate the model, e.g. "param1=1; param2=2"',
```

Comment:
I'm not sure `comma`, `colon` which one better. What do ppl usually use in ml area?

Reply:
This is defined by ourselves. For example, we can use either `--model_params="a=1；b=2" ` or `--model_params="a=1, b=2" `. But using colon as separate will save us from writing a bug-free parser that parses string like `"c=["a", "b"], d=1"` where we cannot simply use `split(",")` since this string contains too many commas.